appveyor_helpers.ps1

```diff
@@ -6,6 +6,22 @@ if (!$env:APPVEYOR)
     $LOCAL_DEBUG_RUN = 1
 }
 
+#
+# Returns true if the current build is triggered by a Pull Request on a specific configuration
+#
+# Remark: We should only run PR-CA once, but Appveyor does not offer a mechanism to differentiate between runs from the same matrix. 
+# As a workaround, we set an extra variable (PR-CA) in the matrix. We also check the os input to not run PR-CA for each "os" 
+#
+function IsPRCABuild
+{
+    if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:PR_CA -and ($env:os -eq "Visual Studio 2015"))
+    {
```

Comment:
APPVEYOR_PULL_REQUEST_NUMBER  is set on PR builds
PR_CA and os are part of the matrix and I use them to differentiate between runs